rotkehlchen/chain/ethereum/modules/liquity/trove.py

```diff
@@ -207,14 +207,15 @@ def __init__(
 
     def get_positions(
         self,
-        addresses: List[ChecksumEthAddress],
+        addresses_list: List[ChecksumEthAddress],
     ) -> Dict[ChecksumEthAddress, Trove]:
         contract = EthereumContract(
             address=LIQUITY_TROVE_MANAGER.address,
             abi=LIQUITY_TROVE_MANAGER.abi,
             deployed_block=LIQUITY_TROVE_MANAGER.deployed_block,
         )
-
+        # make a copy of the list to avoid modifications in the list that is passed as argument
+        addresses = list(addresses_list)
```

Comment:
In general do not modify passed lists. nitpick here: `addressed_list.copy()` is imo a much more readable syntax to soft-copy a list.